modules/store/src/action_creator.ts

```diff
@@ -138,7 +138,7 @@ export function union<
 
 function defineType<T extends string>(
   type: T,
-  creator: Creator
+  creator: Creator & any
```

Comment:
This needs some type fixes

Reply:
I don't know if we can fix this.
We could type assert the return, which is better than the `& any` imho:

```ts
  return Object.defineProperty(creator, 'type', {
    value: type,
    writable: false,
  }) as ActionCreator<T>;
```